3회차/문재윤/3_파리퇴치.py

```diff
@@ -1,3 +1,32 @@
 import sys
 
 sys.stdin = open("_파리퇴치.txt")
+T = int(input())
+
+
+for tc in range(1, T + 1):
+   
+    N, M = map(int, input().split())
+
+
+    mat = [list(map(int, input().split())) for _ in range(N)]
+
+   
+    fly = []
+
+    for i in range(N-M+1):
+        for j in range(N-M+1):
+            total = 0
+            for ci in range(M):
+                for cj in range(M):
+                    if i + ci in range(N) and j + cj in range(N):
+                        total += mat[i + ci][j + cj]           
```

Comment:
깔끔하네요! 👍 17~18번째 줄에서 `range(N-M+1)`까지 탐색을 잘 해주셨습니다.
추가로, 범위 밖으로 나가는 경우는 없어서 22번째 줄의 `if` 문 없이도 잘 작동한답니다 😄 
```suggestion
    for i in range(N-M+1):
        for j in range(N-M+1):
            total = 0
            for ci in range(M):
                for cj in range(M):
                    total += mat[i + ci][j + cj]           
```